vendor/github.com/opencontainers/runtime-tools/generate/seccomp/seccomp_default_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
Just notice this (it was already there), but we're using the seccomp package from runtime-tools somewhere? Does that mean that we're using two distinct profiles? (and if so, is that a problem?)